README.md

```diff
@@ -107,14 +107,13 @@ Other commonly-used configuration options are:
 "mcuboot.slot-size",
 "mcuboot.scratch-address",
 "mcuboot.scratch-size",
-"mcuboot.header-size",
 "mcuboot.max-img-sectors",
 "mcuboot.read-granularity",
 ```
 
 Many of these have been mentioned previously. 
 
-**NOTE:** It is important to ensure the `mcuboot` configuration parameters are **the same** for the bootloader and the bootable application!
+**NOTE:** It is important to ensure the `mcuboot` configuration parameters are **the same** for the bootloader and the bootable application! Also `mcuboot.header-size` has been deprecated as of mcuboot v1.7 and onward.
```

Comment:
Is there something we can use instead of mcuboot.header-size?

Reply:
We don't need to set `mcuboot.header-size` in mbed_app.json no more as of mcuboot v1.8.0 or earlier. We only need to note down the planned header-size and pass it to imgtool when signing built images.

Reply:
Could you update the docs to reflect this? Thanks!

Reply:
I already removed `mcuboot.header-size` from `Other commonly-used configuration options are` section. This confused me a little when I first started working on mcuboot.